source/clear-linux/tutorials/multi-boot/multi-boot-rhel.rst

```diff
@@ -134,15 +132,15 @@ Install Red Hat\* Enterprise Linux 7.4 Beta
 
       .. note:: The root partition UUID used below is unique to this example.
 
-         .. code-block:: console
+         .. code-block:: bash
 
-            # cd /boot/efi/loader/entries
+            sudo cd /boot/efi/loader/entries
 
-            # vi redhat.conf
+            sudo vi redhat.conf
 
-      Add the following lines to :file:`redhat.conf`
+      Add the following lines to the :file:`redhat.conf` file:
 
-      .. code-block:: console
+      .. code-block:: bash
```

Comment:
This should be "code-block:: console" instead "bash" because it's not a command-line output.